ui/src/app/views/registration-dashboard.tsx

```diff
@@ -213,24 +227,71 @@ export class RegistrationDashboard extends React.Component<RegistrationDashboard
     }
   }
 
+  async setAllModulesBypassed(isBypassed: boolean) {
```

Comment:
nit: rename to `setAllModulesBypass` (not to necessarily assume we are setting bypassed=true)